### ISSUE TITLE
[TOTK] Invalid voice index error followed by complete yuzu crash

### ISSUE THREAD

**Issue by noderat** [yuzu-emu/yuzu#10673]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1462 / Early Access 3651

### Description of Issue

Explosions such as bombs, lightning, cannon, etc will occasionally result in a complete crash of Yuzu. Last lines in log indicate "Invalid voice info index" and this error does not appear in logs where the crash does not happen. May be related to #9335 and #10382
```
[7417.857276] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[7451.654564] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0241
[7451.654566] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0260
[7458.234255] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 02A6
[7458.234258] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 02A4
```

Happens in mainline, tested most recently under Early Access to confirm issue is still present in latest build.

### Expected Behavior

Explosions will not cause a complete crash.

### Reproduction Steps

Fuse bomb to x5 bow and aim downwards and fire, then fuse again and aim slightly upwards multiple times until crash. Intermittent, will sometimes work without issue, other times will crash. Some crashes have happened within less than 10 minutes after clean reboot and start of Yuzu, other times hours into a play session.

### Log File

[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/11692695/yuzu_log.zip)

### System Configuration

CPU: AMD Ryzen 7800X3D
GPU/Driver: AMD Radeon 7900XT (20GB) (Tested with drivers 23.4.3 and 23.5.2)
RAM: 64GB
OS: Windows 11 (22H2)

**Comment by Kelebek1** [comment]
Can you post your save to reproduce this?

**Comment by noderat** [comment]
Sure, but crash happens even with a new game. 
[slot_03.zip](https://github.com/yuzu-emu/yuzu/files/11699494/slot_03.zip)

**Comment by Squall-Leonhart** [comment]
The audio issue is a misdirect, its not the cause of the crash, you have a bad game image or the update installed has corrupted.

**Comment by noderat** [comment]
I'll redump my US retail cartridge and I'll try to reproduce using stock 1.0.0 version. Would a my game image still pass signature checks using NX Game Info even if it's a bad dump?

I have my yuzu data and game directories syncing (rsync) to my Steam Deck and I do not have this crash (or any crash for that matter) on the Steam Deck hardware with the exact same update install and XCI image (I reconfirmed checksums just to be sure). The only difference is hardware and a post-sync script that auto-adjusts the game config from using 60hz static mod @ 2x and surround audio on desktop to 30hz static mod @ 1x and stereo audio on Steam Deck (simple sed find/replace script)

After I redump my cart I'll try to reproduce the crash again using the same settings and mod configuration as my Steam Deck.

**Comment by Pancakse** [comment]
I seem to have been getting the same crash on yuzu in TOTK. For me it usually occurs after throwing bombs. here's the end of my log:

2771.215902] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2806.374036] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA6000 (start address = 0x00000010D7A9B000, size = 61440)
[2806.374041] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA7000 (start address = 0x00000010D7A9B000, size = 61440)
[2806.374042] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA8000 (start address = 0x00000010D7A9B000, size = 61440)
[2806.374043] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA9000 (start address = 0x00000010D7A9B000, size = 61440)
[2806.699347] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2810.300416] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2834.143871] Render.Vulkan <Info> video_core\renderer_vulkan\vk_pipeline_cache.cpp:CreateGraphicsPipeline:579: 0x0ed628ecb9c83cbe
[2892.440940] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA6000 (start address = 0x00000010D7A99000, size = 69632)
[2892.440985] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA7000 (start address = 0x00000010D7A99000, size = 69632)
[2892.440985] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA8000 (start address = 0x00000010D7A99000, size = 69632)
[2892.440988] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA9000 (start address = 0x00000010D7A99000, size = 69632)
[2915.916864] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA6000 (start address = 0x00000010D7A9B000, size = 61440)
[2915.916869] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA7000 (start address = 0x00000010D7A9B000, size = 61440)
[2915.916870] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA8000 (start address = 0x00000010D7A9B000, size = 61440)
[2915.916871] HW.Memory <Error> core\memory.cpp:operator ():239: Unmapped ReadBlock @ 0x00000010D7AA9000 (start address = 0x00000010D7A9B000, size = 61440)
[2916.099344] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2933.399392] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2952.795888] Service.FS <Warning> core\hle\service\filesystem\fsp_srv.cpp:Commit:480: (STUBBED) called
[2976.713947] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.713951] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0284
[2976.720193] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.720197] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0284
[2976.723908] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.723912] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0284
[2976.740714] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.744067] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.750750] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.753877] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.760232] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.764001] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2976.764004] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0284
[2976.764004] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 01E5
[2997.563904] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2997.563908] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0247
[2997.563908] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0269
[2997.579280] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0272
[2997.579284] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0247
[2997.579284] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0269
[2997.579285] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 021E
[2997.579285] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 025C
[2997.579286] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0251
[2997.579286] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0265
[2997.579286] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0281
[2997.579287] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 027A
[2997.579287] Service.Audio <Error> audio_core\renderer\voice\voice_context.cpp:GetInfo:48: Invalid voice info index 0233

**Comment by liamwhite** [comment]
fixed by #11408
